clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerRebalanceListener.java

```diff
@@ -144,9 +144,9 @@ public interface ConsumerRebalanceListener {
     void onPartitionsRevoked(Collection<TopicPartition> partitions);
 
     /**
-     * A callback method the user can implement to provide handling of customized offsets on completion of a successful
-     * partition re-assignment. This method will be called after the partition re-assignment completes and before the
-     * consumer starts fetching data, and only as the result of a {@link Consumer#poll(java.time.Duration) poll(long)} call.
+     * A callback method the user can implement to be notified when partitions are assigned to this consumer.
```

Comment:
I think that "provide handling" is fine, especially since it was used for `onPartitionsRevoked()`.

```suggestion
     * A callback method the user can implement to provide handling when partitions are assigned to this consumer.
```